projects/schematics/src/shared/utils/file-utils_spec.ts

```diff
@@ -176,20 +203,313 @@ describe('File utils', () => {
     });
   });
 
+  describe('isCandidateForConstructorDeprecation', async () => {
+    it('should return false if the inheritance condition is not satisfied', () => {
+      const content = `
```

Comment:
Can we go with consts before tests (as in `update-cms-component-state_spec.ts`)? Just for more consistency

Reply:
Done for all comments.